### PR TITLE
Fix for file upload not working

### PR DESCRIPTION
**Pull request by CharlieHawker**
As described in issue #9, this fixes deprecated code preventing file fields being submitted to Wufoo correctly.

### CODE REVIEW

**Review by friendofdog — APPROVED** [review]

**Comment by mrself** [comment]
Why have not this been merged?

**Comment by CharlieHawker** [comment]
My guess is because the project is (very) stale @mrself - check out the commit history; not updated since 2010, other than to add the MIT license!

Feel free to use my fork if you want this feature

**Comment by mrself** [comment]
@CharlieHawker I see now, agree with you. Now I am not surprised I hasn't composer support. Maybe wufoo should share write access to some people wishing improve it.

I could add composer support.

**Comment by CharlieHawker** [comment]
Am closing this PR as nobody seems to want to merge despite approval by @friendofdog - never mind!

**Pull request closed without merging**
